packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthErrorHandlerTest.kt

```diff
@@ -366,7 +366,7 @@ class AuthErrorHandlerTest {
     @Test
     fun sessionUnavailableOfflineException() {
         // Arrange
-        val expectedCode = "AuthException"
+        val expectedCode = "SessionUnavailableOfflineException"
```

Comment:
We previously were not mapping this code, but I think we want to given that we handle this code on the Dart side.